source/conf.py

```diff
@@ -48,6 +49,7 @@
 # ones.
 extensions = [
     "sphinx.ext.autodoc",
+    "sphinx.ext.intersphinx",
```

Comment:
@aroberge using `intersphinx` may be an overkill - used that to reference `OSError` and `ConnectionError` to indicate those are real :-) Say the word and I'll remove that.